CUSTOM_IMAGE.md

```diff
@@ -0,0 +1,76 @@
+Building a Custom Spark Image
+=============================
+
+Overview
+--------
+To build a custom Spark image, with a custom version of Spark or other libraries, 
+such as Hadoop, Python, or R, follow the steps below. This build does the following:
+1. Builds a Docker image containing Spark and other libraries.
+1. Builds a DC/OS package (also called a "universe") based on this Docker image.
+1. Uploads this package to Amazon S3.
+
+Prerequisites
+-------------
+* [AWS Command Line Interface](https://aws.amazon.com/cli/) (CLI)
+* The [jq](https://github.com/stedolan/jq) JSON parser
+
+Custom Spark Version, Hadoop Version
```

Comment:
I don't think we want to support custom spark versions.  Maybe after we merge our custom commits into upstream, but with separate apache/spark and mesosphere/spark repos, this is just going to confuse people.  They're going to try to build apache/spark, and complain when custom DC/OS features don't work.